src/main/java/com/github/soerxpso/xpspawners/listeners/MobSpawnListener.java

```diff
@@ -21,13 +20,12 @@ public MobSpawnListener() {
 	@EventHandler(priority = EventPriority.LOW, ignoreCancelled = false)
 	public void onMobSpawn(SpawnerSpawnEvent e) {
 		e.setCancelled(true);
-		e.getSpawner().setDelay(ConfigManager.getHarvestInterval());
+		e.getSpawner().setDelay(Integer.MAX_VALUE);
```

Comment:
Not sure I understnd the change to MAX_VALUE here -- what triggers the XP drop now?